Specs/Widgets/InfoBox/InfoBoxViewModelSpec.js

```diff
@@ -14,6 +16,13 @@ defineSuite([
         expect(viewModel.maxHeightOffset(0)).toBeDefined();
     });
 
+    it('sets title', function() {
```

Comment:
I am just going to remove this test, as all the necessary tests are already in InfoBoxSpec